src/components/SortableCommentList.js

```diff
@@ -140,7 +145,7 @@ class SortableCommentList extends Component {
 
         {canComment && <CommentForm
           hearingId={hearingId}
-          onPostComment={this.onPostHearingComment.bind(this)}
+          onPostComment={!onPostComment ? this.this.onPostHearingComment.bind(this) : onPostComment}
```

Comment:
For some reason, in the subsections, the comment form does not allow selecting a nickname at all. In the main hearing, the nickname box is shown. Is `canSetNickname` set right?